moveit_ros/hybrid_planning/local_planner/local_constraint_solver_plugins/src/forward_trajectory.cpp

```diff
@@ -150,7 +151,7 @@ ForwardTrajectory::solve(const robot_trajectory::RobotTrajectory& local_trajecto
         {
           num_iterations_stuck_ = 0;
           prev_waypoint_target_ = nullptr;
-          feedback_result.feedback = "local_planner_stuck";
+          feedback_result.feedback = FeedbackEnumToString(LocalFeedbackEnum::LOCAL_PLANNER_STUCK);
```

Comment:
if you change the name of the function this just becomes:

```suggestion
          feedback_result.feedback = toString(LocalFeedbackEnum::LOCAL_PLANNER_STUCK);
```